File: JAVA/02_Recursion/Lecture09/pdis.java
package Lecture09;

public class pdis {

	public static void main(String[] args) {
		pdiss(7);
	}
	public static void pdiss(int n) {
		if(n == 0) {
			return;
		}
		if(n % 2 != 0) {
			System.out.print(n + " ");
		}
		pdiss(n - 1);
		if(n % 2 == 0) {
			System.out.print(n + " ");
		}
	}

}

File: C++/DS1/Lec9/pdis.cpp
#include <iostream>
using namespace std;
void pdis(int n){
    if(n == 0){
        return;
    }
    if(n % 2 == 1)
    cout << n << " ";
    pdis(n - 1);
    if(n % 2 == 0)
    cout << n << " ";
}
int main(){
    pdis(7);
    cout << endl;
    return 0;
}